var/spack/repos/builtin/packages/vcftools/package.py

```diff
@@ -25,6 +25,7 @@ class Vcftools(AutotoolsPackage):
 
     depends_on("perl", type=("build", "run"))
     depends_on("zlib-api")
+    depends_on("pkg-config")
```

Comment:
Does this need to be `pkg-config` or could it be satisfied with `pkgconf`?  

If the implementation doesn't matter, this could be a dependency on the virtual package `pkgconfig`.

Reply:
When I edit the recipe and replace this with `pkgconfig` and `spack concretize -f` this does not change the spack.lock file for me so I believe that it can be `pkgconf`.

Shall I make that change ?

Reply:
Yes, since specifying the virtual could help others who want to build the package.